py/build.gradle

```diff
@@ -3,6 +3,13 @@ apply from: "$rootDir/gradle/utils.gradle"
 ext {
     FS = File.separator
     FPS = File.pathSeparator
+
+    def command = ["python", "py/pythonVersion.py"]
```

Comment:
Yes, I was having problems while invoking the contents of pythonVersion.py directly from the gradle script as above using the `-c` option, not sure if I was doing something wrong but I didn't manage to get it to work :-/